### ISSUE TITLE
Shouldn't overwrite old image by default

### ISSUE THREAD

**Issue by fnielsen** [danmichaelo/croptool#48]
I cannot see why images should be overwritten by default.

**Comment by danmichaelo** [comment]
The main use case I had in mind when writing this tool was removing borders. In most of the cases I've looked at, the borders didn't add educational value ([example](https://commons.wikimedia.org/wiki/File:Davidlbazelon.jpg)) and then there's no reason to keep the bordered versions as a separate file, since multiple versions of the same file opens for inconsistency (description or categorization updated for one version, not the other), pollutes categories and makes retrieval more difficult. Some exceptions are tagged with https://commons.wikimedia.org/wiki/Template:Border_is_intentional

I guess people use CropTool for all kinds of things, so my default might not be the best for others. That's why I added the link to the overwrite policy.
